soroban-env-host/src/host/declared_size.rs

```diff
@@ -398,7 +397,19 @@ mod test {
             expect!["104"].assert_eq(size_of::<LedgerKey>().to_string().as_str());
         }
 
+        #[rustversion::before(1.77)]
```

Comment:
@graydon I see that you had to do something similar [here](https://github.com/stellar/rs-soroban-env/commit/1999a9cf16a53137da69287a5e8c1c0c76d370f6#diff-bd7d71cced65f745b19e26f023160e33773531af294c0e6fb036031f6d4f237f), but in your case the struct got smaller, not bigger. Is this a metering concern? Both from an undercharging perspective and divergence. I'm guessing not because we already have a difference between architectures, but I'm curious what we need to think about when we see this happen.

Reply:
This should be fine. 8 bytes under-charging per host object should be within acceptable tolerance. There plenty other areas where host object charges are accounted for (mem alloc, visit object etc.). 
These tests are purely for informational use, to notice these changes as versions evolve and take actions when necessary. If a number go down, nothing else is required. If it goes up significantly, we will need to branch a protocol for its declared size.